<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BulkEditDialog</class>
 <widget class="QDialog" name="BulkEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulk Edit Items</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <widget class="QWidget" name="itemsTab">
      <attribute name="title">
       <string>Items</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QTreeView" name="itemsTree">
         <property name="styleSheet">
          <string notr="true">QTreeView{
	background: transparent; 
	border:0;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>0</number>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonBoxItems">
         <item>
          <widget class="QPushButton" name="buttonRemoveItem">
           <property name="toolTip">
            <string>Remove from Bulk Edit</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelItemsCount">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;0 Items&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="relationsTab">
      <attribute name="title">
       <string>Relations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QTreeView" name="relationsTree">
         <property name="styleSheet">
          <string notr="true">QTreeView{
	background: transparent; 
	border:0;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>0</number>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonBoxRelations">
         <item>
          <widget class="QPushButton" name="buttonAddRelation">
           <property name="toolTip">
            <string>Add Relation Argument</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRemoveRelation">
           <property name="toolTip">
            <string>Remove Selected Row</string>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonClearRelations">
           <property name="toolTip">
            <string>Clear All Rows</string>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BulkEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BulkEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
